src/main/java/org/gridsuite/study/server/service/shortcircuit/ShortCircuitService.java

```diff
@@ -291,4 +322,30 @@ public void assertShortCircuitAnalysisNotRunning(UUID nodeUuid) {
             throw new StudyException(SHORT_CIRCUIT_ANALYSIS_RUNNING);
         }
     }
+
+    public static final List<VoltageRange> CEI909_VOLTAGE_PROFILE = List.of(
+            new VoltageRange(20.0, 224.99, 1.1),
+            new VoltageRange(225.0, 379.99, 1.09),
+            new VoltageRange(380.0, 420.0, 1.05)
+    );
+
+    public static ShortCircuitParametersInfos toShortCircuitParametersInfo(ShortCircuitParametersEntity entity) {
+        Objects.requireNonNull(entity);
+        return ShortCircuitParametersInfos.builder()
+                .predefinedParameters(entity.getPredefinedParameters())
+                .parameters(fromEntity(entity))
+                .voltageRangesInfo(CEI909_VOLTAGE_PROFILE)
```

Comment:
Same. Why CEI909_VOLTAGE_PROFILE ?

Reply:
As  discussed before voltageRangesInfo must contains the voltageRanges related to the CONFIGURED (aka CEI909) initial voltage profile mode.these values are used in Front end.

Reply:
Yes ok with that. Just a rename is required then. As you said, it's the voltage ranges for CEI909. So it should be cei909VoltageRanges and not voltageRangesInfo. Fuzzy names for variables or functions will generate confusion. And after few commits, we won't understand anything

Reply:
Name changed